lib/hmmlearn/tests/__init__.py

```diff
@@ -85,3 +85,12 @@ def compare_variational_and_em_models(variational, em, sequences, lengths):
     vi_obs, vi_states = variational.sample(100, random_state=42)
     assert np.all(em_obs == vi_obs)
     assert np.all(em_states == vi_states)
+
+
+def vi_uniform_startprob_and_transmat(model, lengths):
+    nc = model.n_components
+    model.startprob_prior_ = np.array([1/nc]*nc)
+    model.startprob_posterior_ = np.array([1/nc]*nc) * len(lengths)
+    model.transmat_prior_ = np.full((nc,nc), 1/nc)
+    model.transmat_posterior_ = np.full((nc,nc), 1/nc) * sum(lengths)
```

Comment:
At least add a comma in (nc, nc).  I would also rewrite the first two lines as np.full(nc, 1/nc) for consistency with the ones below (`np.array([1/nc]*nc) * len(lengths)` also reads slightly confusingly because the two `*` next to one another have different semantics) but don't feel extremely strongly about it.